agent/stats/utils.go

```diff
@@ -35,9 +38,8 @@ func nan32() float32 {
 // dockerStatsToContainerStats returns a new object of the ContainerStats object from docker stats.
 func dockerStatsToContainerStats(dockerStats *docker.Stats) (*ContainerStats, error) {
 	// The length of PercpuUsage represents the number of cores in an instance.
-	numCores := uint64(len(dockerStats.CPUStats.CPUUsage.PercpuUsage))
-	if numCores == 0 {
-		seelog.Debug("Invalid container statitistics reported, got number of cores = 0")
+	if len(dockerStats.CPUStats.CPUUsage.PercpuUsage) == 0 {
```

Comment:
From the stress [output](https://gist.github.com/richardpen/cb80f285ab558e79a4a8ea0a116cf629), if `len(dockerStats.CPUStats.CPUUsage.PercpuUsage)` is `0`, the data returned is useless, so it should be ignored.